build/util/LASTCHANGE.dummy

```diff
@@ -1 +1,2 @@
-LASTCHANGE=0000000000000000000000000000000000000000-0000000000000000000000000000000000000000
```

Comment:
Is this change intentional?

Reply:
Yes, this is a hotfix recommended in b/348690792 to get net_unittests to run. Will remove if it doesn't end up changing anything.

Reply:
Removed